src/BattleCalculator-Freewar/BattleCalculator-Freewar.user.js

```diff
@@ -92,8 +122,10 @@ function processElement(cellElement) {
 			if (lifeLoss >= critLifeThreshold || npcName == 'Undaron') {
 				$(npcFastAttackElement).css('color', '#E7971F');
 			}
-			$(npcFastAttackElement).append(' (-' + lifeLoss + ' LP)');
+		//	$(npcFastAttackElement).append(' (-' + lifeLoss + ' LP)');
 			$(npcNameElement).addClass('processedNPC knownNPC');
+            $(npcFastAttackElement).css('color', '#006400');
+            $(npcFastAttackElement).append(' - LS: ' + lifeLoss + ' - NL: ' + (playerExpectedLife - lifeLoss));
```

Comment:
imo `LS` and `NL` is not really intuitive. maybe go for:
```java
(-15 LP, =230 LP)
```

Reply:
good point, thx

changed

![Unbenannt](https://user-images.githubusercontent.com/14137153/217653147-3b5b29a3-87fe-4f7a-b9a9-10e7c4e63d65.PNG)